### DISCUSSION TITLE
v0.7.0

### DISCUSSION THREAD

**Discussion by an-lee** [ZuodaoTech/everyone-can-use-english#1189] [Announcements]
## What's Changed
* Fix transcription by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1137
* Upgrade deps by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1148
* Support document(epub) by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1160
* upgrade deps by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1167
* Improve settings by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1174
* Fix document delete by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1176
* Improve: UI & code by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1179
* Improve audio/video adding & display by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1181
* Improve transcription by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1182
* Customize title bar by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1184
* Upgrade deps by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1185
* Macos  support whisper cpp by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1186
* Update sidebar by @an-lee in https://github.com/ZuodaoTech/everyone-can-use-english/pull/1188

**Full Changelog**: https://github.com/ZuodaoTech/everyone-can-use-english/compare/v0.6.1...v0.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ZuodaoTech/everyone-can-use-english/releases/tag/v0.7.0'>v0.7.0</a>.</em>

**Comment by woshixiaocaomao** [comment]
上传本地文档后费用都是怎么收取的，怎么感觉账户里的数字往下掉？还有只能看到充值记录，看不到具体消费记录。

**Reply by an-lee** [reply]
上传文档不收费。只有翻译、生成语音会收费。软件设置 / 账户设置 里能看到扣费的具体情况。

**Reply by WeiMao33** [reply]
充值2美元，选Stripe支付，选中支付宝，然后发现支付宝要收18.8元人民币，手续费有点高啊，可以尝试直接接入支付宝吗？

**Reply by an-lee** [reply]
额外的都是法币渠道费，没办法。多充点的话渠道费平均下来会便宜点。另外， 有条件的也可以选择虚拟货币充值，没有渠道费。

**Reply by WeiMao33** [reply]
谢谢

**Comment by taosz** [comment]
求助 ！今天始终无法连上服务器

platform: win32 x64 10.0.19045
version: 7.3

尝试了以下步骤：
1. 从https://enjoy.bot切换到https://api.getenjoyapp.com/
2. 设置代理地址为http://localhost:7890 (已启动clash for windows并监听到7890端口，可正常访问google)
3. 直接在浏览器中打开 https://enjoy.bot 和 https://api.getenjoyapp.com, 多次尝试都在 pending 状态，有时可以访问成功， 但是需要 loading 很久
4. 在 【enjoy app -> 软件设置 -> 高级设置 】 中，网络状态如下图
![image](https://github.com/user-attachments/assets/5dac2a16-ce96-4bfe-a900-6fa3378b2339)

**Reply by taosz** [reply]
我尝试删除个人数据，重新登录，但是出现了这个问题

![image](https://github.com/user-attachments/assets/e9b33469-2386-49b9-934f-9832dba06d8e)

**Reply by taosz** [reply]
有时可以走到下一步，像下面这样，随后就无法继续了
![image](https://github.com/user-attachments/assets/8f777801-8186-4255-be2c-6b4f937aa0b4)

**Reply by an-lee** [reply]
今天早上云服务器故障了，所以服务器连接不稳定。应该快修复好了。

**Comment by JingIcey** [comment]
使用AI反应功能时，文本无法复制粘贴
